spec/mws-rb/api/merchant_fulfillment_spec.rb

```diff
@@ -0,0 +1,24 @@
+require 'spec_helper'
+
+describe MWS::API::FulfillmentOutboundShipment do
```

Comment:
Class name isn't correct here. As well as `fulfillment_outbound_shipment` variable in the spec.